.github/workflows/test.yml

```diff
@@ -4,14 +4,14 @@ on: [push, pull_request]
 jobs:
   test:
     name: Test
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
     steps:
-      - name: Set up Go 1.20
-        uses: actions/setup-go@v1
+      - name: Set up Go 1.24
+        uses: actions/setup-go@v5
         with:
-          go-version: 1.20
+          go-version: '1.24'
```

Comment:
Why don't we modify the go version in the go.mod as well?
By doing so, we can set `go-version-file: go.mod` instead of `go-version`.

Reply:
Oh, it seems to be better to keep the current version not to force applications that depend on this library to upgrade Go version.

Reply:
Thank you for your review. Yes! Regarding the `go` version in the `go.mod` file, I believe using a lower version is better.